File: PRAK205-2510817110006-Muhammad Fadhil Lesmana.c
#include <stdio.h>

#define PI 3.14159265358979323846

int main() {
    float r, t;
    
    //Input
    scanf("%f", &r);
    scanf("%f", &t);
    
    //Perhitungan
    float volume = PI * r * r * t;
    float luas = 2 * PI * r * (r + t);
    float keliling = 2 * PI * r;

    //Output dengan 2 angka di belakang koma
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);
    return 0;
}

File: PRAK205-2510817110006-Muhammad Fadhil Lesmana.py
import math

#Input
r = float(input())
t = float(input())

#Perhitungan
volume = math.pi * r * r * t
luas = 2 * math.pi * r * (r + t)
keliling = 2 * math.pi * r

#Output dengan 2 angka di belakang koma
print("Volume = {:.2f}".format(volume))
print("Luas = {:.2f}".format(luas))
print("Keliling = {:.2f}".format(keliling))
